web-marketplace/src/pages/Projects/Projects.tsx

```diff
@@ -126,11 +133,7 @@ export const Projects: React.FC<React.PropsWithChildren<unknown>> = () => {
     setUseOffChainProjects(undefined);
   };
 
-  if (
-    loading ||
```

Comment:
why removing projects loading from here?

Reply:
Because it was closing the filter panel each time a filter was applied. I'm not sure if it's a regression or not but it wasn't great UX-wise, this fixes it, and still has an initial loader based on other calls.